app/src/bcsc-theme/features/home/Home.tsx

```diff
@@ -58,7 +67,7 @@ const Home: React.FC<HomeProps> = ({ navigation }) => {
         <ActivityIndicator size={'large'} style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }} />
       ) : (
         <>
-          <NotificationBannerContainer />
+          <NotificationBannerContainer onManageDevices={handleManageDevices} />
```

Comment:
question: Why do we need to pull `handleManageDevices` into the parent?

Reply:
The navigation callback required is different depending on which stack it's being used in